clients/spark/core/src/test/resources/parser-test/sst_files_generator.go

```diff
@@ -65,7 +65,10 @@ func writeSstWithBadMagic() {
 	if err != nil {
 		panic(err)
 	}
-	os.Truncate(testFileName+".sst", fi.Size()-MagicLengthBytes/2)
+	err = os.Truncate(testFileName+".sst", fi.Size()-MagicLengthBytes/2)
```

Comment:
Rather more than spelling here, I think. Consider updating the PR title.

Reply:
You are right, it started with my spelling linter and from there other linters started to pull me in.